src/helpers.ts

```diff
@@ -215,13 +212,18 @@ export const cutAddress = (address: string): string => {
   if (clean.length === 56) {
     const v3 = "[a-z2-7]{56}";
     if (new RegExp(`${v3}`).test(clean)) {
-      return clean.substr(0, 20) + '...' + clean.substr(36);
+      return clean.substr(0, 20) + "..." + clean.substr(36);
     }
   }
 
   if (clean.length > 42) {
-    clean = clean.substr(0, 38) + '...';
+    clean = clean.substr(0, 38) + "...";
   }
 
   return clean;
 };
+
+export const countDecimals = (value: number): number => {
+  if (Math.floor(value) === value) return 0;
+  return value.toString().split(".")[1].length || 0;
```

Comment:
Does toString always use a decimal? Or does it use globalization?

Reply:
The `Amount` property do not accept `","` as an input, since Amount is a `number`.

```javascript
Number("1,0234");  // will return NaN
Number("1,0234.120");  // will return NaN
```

Instead of using custom code, I decide to use [countDecimalPlaces()](https://github.com/palantir/blueprint/blob/release/3.17.x/packages/core/src/common/utils.ts#L208), since is already available and tested.